### ISSUE TITLE
Testers and developers wanted for Linux, Windows, macOS & web !

### ISSUE THREAD

**Issue by juliansteenbakker** [mogol/flutter_secure_storage#263]
With the release of version 5.0.0-beta.1 we have added beta support for Linux, Windows, macOS and web!

It would be great if some people can test and improve these specific platform implementations as i don't have the time to do this all by myself.

The plugin is now separated into different projects for each platform which is called a federated plugin. You can read more about [how this works here](https://docs.google.com/document/d/1LD7QjmzJZLCopUrFAAE98wOUQpjmguyGTN2wd_89Srs).

**Comment by jhancock4d** [comment]
Heads up, I'm putting this to use in our app that is being tested for production release soon. So I should have lots of feedback, especially on the web implementation.

**Comment by kennethnym** [comment]
Just a heads up, I am working on implementing `readAll` and `deleteAll` on Windows.

**Comment by Nealsoni00** [comment]
I get a crash when trying to get a value stored in the store on windows. I call this method in main after `WidgetsFlutterBinding.ensureInitialized()`. The app crashes without throwing an error or anything :(. 

I don't think anything is stored in the secure store, but i comment out the line, the app runs properly. Any advice would be helpful! 

![image](https://user-images.githubusercontent.com/7013191/128948802-6346c878-212a-4011-a85e-8a0dd660d9e7.png)

**Comment by Nealsoni00** [comment]
replacing the error sequence in `Read` with
```
  if (error == ERROR_NOT_FOUND) {
        return std::string("ERROR");
      }
```
prevents the crash. 

Returning `nullopt` is 100% better if possible but I couldn't get that to work (noob in C++).

**Comment by kennethnym** [comment]
In the current implementation, the private `Read` method does return `nullopt` if the key doesn't exist, however, I didn't take this into account at call site, meaning I forgot to do a `has_value()` check before accessing `value()` and returning that value to the method channel. Should be an easy fix. Thanks for reporting.

**Comment by Jon-Salmon** [comment]
@juliansteenbakker I believe I have closed out all the open issues relating to the windows implementation. If I've missed anything or you would like me to test anything in particular please feel free to tag me.

**Comment by juliansteenbakker** [comment]
@Jon-Salmon Thank you for your work! I have released all your improvements in v5.0.0
